src/sparse_blas/backends/cusparse/CMakeLists.txt

```diff
@@ -43,6 +43,24 @@ target_include_directories(${LIB_OBJ}
 
 target_compile_options(${LIB_OBJ} PRIVATE ${ONEMATH_BUILD_COPT})
 
+if (NOT "${ONEMATH_SYCL_IMPLEMENTATION}" STREQUAL "adaptivecpp")
+  target_compile_options(ONEMATH::SYCL::SYCL INTERFACE
+    -fsycl-targets=nvptx64-nvidia-cuda -fsycl-unnamed-lambda
+  )
+  target_link_options(ONEMATH::SYCL::SYCL INTERFACE
+    -fsycl-targets=nvptx64-nvidia-cuda
+  )
+
+  if (DEFINED CUDA_TARGETS AND NOT "${CUDA_TARGETS}" STREQUAL "")
+    target_compile_options(ONEMATH::SYCL::SYCL INTERFACE
+      -Xsycl-target-backend --cuda-gpu-arch=${CUDA_TARGETS}
+    )
+    target_link_options(ONEMATH::SYCL::SYCL INTERFACE
+      -Xsycl-target-backend --cuda-gpu-arch=${CUDA_TARGETS}
+    )
+  endif()
+endif()
+
```

Comment:
this is fine, it is same update as for BLAS, for CUDA + SYCL linking, so no reason to object